code/modules/projectiles/guns/projectile.dm

```diff
@@ -259,7 +259,7 @@
 		return ..()
 	if (loaded.len || stored_magazine || refuse.len)
 		if (load_method == SPEEDLOADER)
-			if(!gun_flags & CHAMBERSPENT)
+			if(!(gun_flags & CHAMBERSPENT))
```

Comment:
This changes behavior in theory, but in practice I don't think there's any gun in the game for which anything actually changes except possibly the base `/obj/item/weapon/gun/projectile`